### DISCUSSION TITLE
I can't install Filament on Laravel 11

### DISCUSSION THREAD

**Discussion by syarifulumam** [filamentphp/filament#12248] [Help] [answered]
### Package

Other

### Package Version

3.2.63

### How can we help you?

`composer require filament/filament:"^3.2" -W`

The "3.2" constraint for "filament/filament" appears too strict and will likely not match what you want. See https://getcomposer.org/constraints
./composer.json has been updated
Running composer update filament/filament --with-all-dependencies
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires filament/filament 3.2 -> satisfiable by filament/filament[v3.2.0].
    - filament/filament v3.2.0 requires illuminate/console ^10.0 -> found illuminate/console[v10.0.0, ..., 10.x-dev] but these were not loaded, likely because it conflicts with another require.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

i try 
`composer require filament/filament`

./composer.json has been updated
Running composer update filament/filament
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - filament/filament v3.2.39 requires filament/support v3.2.39 -> satisfiable by filament/support[v3.2.39].
    - filament/filament v3.2.40 requires filament/support v3.2.40 -> satisfiable by filament/support[v3.2.40].
    - filament/filament v3.2.41 requires filament/support v3.2.41 -> satisfiable by filament/support[v3.2.41].
    - filament/filament v3.2.42 requires filament/support v3.2.42 -> satisfiable by filament/support[v3.2.42].
    - filament/filament v3.2.43 requires filament/support v3.2.43 -> satisfiable by filament/support[v3.2.43].
    - filament/filament v3.2.44 requires filament/support v3.2.44 -> satisfiable by filament/support[v3.2.44].
    - filament/filament v3.2.45 requires filament/support v3.2.45 -> satisfiable by filament/support[v3.2.45].
    - filament/filament v3.2.46 requires filament/support v3.2.46 -> satisfiable by filament/support[v3.2.46].
    - filament/filament v3.2.47 requires filament/support v3.2.47 -> satisfiable by filament/support[v3.2.47].
    - filament/filament v3.2.48 requires filament/support v3.2.48 -> satisfiable by filament/support[v3.2.48].
    - filament/filament v3.2.49 requires filament/support v3.2.49 -> satisfiable by filament/support[v3.2.49].
    - filament/filament v3.2.50 requires filament/support v3.2.50 -> satisfiable by filament/support[v3.2.50].
    - filament/filament v3.2.51 requires filament/support v3.2.51 -> satisfiable by filament/support[v3.2.51].
    - filament/filament v3.2.52 requires filament/support v3.2.52 -> satisfiable by filament/support[v3.2.52].
    - filament/filament v3.2.53 requires filament/support v3.2.53 -> satisfiable by filament/support[v3.2.53].
    - filament/filament v3.2.54 requires filament/support v3.2.54 -> satisfiable by filament/support[v3.2.54].
    - filament/filament v3.2.55 requires filament/support v3.2.55 -> satisfiable by filament/support[v3.2.55].
    - filament/filament v3.2.56 requires filament/support v3.2.56 -> satisfiable by filament/support[v3.2.56].
    - filament/filament v3.2.57 requires filament/support v3.2.57 -> satisfiable by filament/support[v3.2.57].
    - filament/filament v3.2.58 requires filament/support v3.2.58 -> satisfiable by filament/support[v3.2.58].
    - filament/filament v3.2.59 requires filament/support v3.2.59 -> satisfiable by filament/support[v3.2.59].
    - filament/filament v3.2.60 requires filament/support v3.2.60 -> satisfiable by filament/support[v3.2.60].
    - filament/filament v3.2.61 requires filament/support v3.2.61 -> satisfiable by filament/support[v3.2.61].
    - filament/filament v3.2.62 requires filament/support v3.2.62 -> satisfiable by filament/support[v3.2.62].
    - filament/filament v3.2.63 requires filament/support v3.2.63 -> satisfiable by filament/support[v3.2.63].
    - filament/filament v0.1.0 requires php ^7.4 -> your php version (8.2.12) does not satisfy that requirement.
    - filament/filament[v1.0.0, ..., v1.13.4] require illuminate/auth ^8.0 -> found illuminate/auth[v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - filament/filament[v2.5.3, ..., v2.9.6] require illuminate/auth ^8.6 -> found illuminate/auth[v8.6.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - filament/filament[v2.9.7, ..., v2.16.65] require illuminate/auth ^8.6|^9.0 -> found illuminate/auth[v8.6.0, ..., v8.83.27, v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - filament/filament v2.15.0 requires filament/notifications v2.15.0 -> found filament/notifications[dev-3.x-alpha, v2.15.1, ..., 2.x-dev, v3.0.0-alpha1, ..., 3.x-dev] but it does not match the constraint.
    - filament/filament[v2.16.66, ..., v2.17.55] require illuminate/auth ^8.6|^9.0|^10.0 -> found illuminate/auth[v8.6.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.4] but these were not loaded, likely because it conflicts with another require.
    - filament/filament v2.17.23 requires filament/tables v2.17.23 -> found filament/tables[dev-3.x-alpha, v1.0.0, ..., v1.10.12, v2.3.6, ..., 2.x-dev, v3.0.0-alpha1, ..., 3.x-dev] but it does not match the constraint.
    - filament/filament[v3.0.0, ..., v3.0.8] require illuminate/auth ^9.0|^10.0 -> found illuminate/auth[v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.4] but these were not loaded, likely because it conflicts with another require.
    - filament/filament[v3.0.9, ..., v3.2.35] require illuminate/auth ^10.0 -> found illuminate/auth[v10.0.0, ..., v10.48.4] but these were not loaded, likely because it conflicts with another require.
    - filament/filament v3.1.38 requires filament/infolists v3.1.38 -> found filament/infolists[dev-3.x-alpha, v3.0.0-alpha1, ..., 3.x-dev] but it does not match the constraint.
    - filament/filament[v3.2.36, ..., v3.2.38] require illuminate/auth ^10.45 -> found illuminate/auth[v10.45.0, ..., v10.48.4] but these were not loaded, likely because it conflicts with another require.
    - filament/support[v3.2.39, ..., v3.2.63] require ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.
    - Root composer.json requires filament/filament * -> satisfiable by filament/filament[v0.1.0, v1.0.0, ..., v1.13.4, v2.5.3, ..., v2.17.55, v3.0.0, ..., v3.2.63].

To enable extensions, verify that they are enabled in your .ini files:
    - C:\xampp\php\php.ini
You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with `--ignore-platform-req=ext-intl` to temporarily ignore these required extensions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require filament/filament:*" to figure out if any version is installable, or "composer require filament/filament:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

composer.json
```{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The skeleton application for the Laravel framework.",
    "keywords": [
        "laravel",
        "framework"
    ],
    "license": "MIT",
    "require": {
        "php": "^8.2",
        "laravel/framework": "^11.0",
        "laravel/jetstream": "^5.0",
        "laravel/sanctum": "^4.0",
        "laravel/tinker": "^2.9",
        "livewire/livewire": "^3.0"
    },
    "require-dev": {
        "fakerphp/faker": "^1.23",
        "laravel/pint": "^1.13",
        "laravel/sail": "^1.26",
        "mockery/mockery": "^1.6",
        "nunomaduro/collision": "^8.0",
        "pestphp/pest": "^2.0",
        "pestphp/pest-plugin-laravel": "^2.0",
        "spatie/laravel-ignition": "^2.4"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi",
            "@php -r \"file_exists('database/database.sqlite') || touch('database/database.sqlite');\"",
            "@php artisan migrate --graceful --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true
}
```

**Comment by leandrocfe** [comment, marked as answer]
Did you install the `ext-intl` php extension?
After installing it, just run `composer require filament/filament`

**Reply by Abdullatif-Erfan** [reply]
This solution worked for me, thanks

**Reply by oscarenriqq** [reply]
Thank you!!

**Comment by PseudoDevs** [comment]
Your Command:
`composer require filament/filament:"^3.2" -W`

Change the composer command to:
`composer require filament/filament:"^3.2.(latest-version-of-filament)" -W`

Sample:
`composer require filament/filament:"^3.2.63" -W`

and enable also the `ext-intl` from your prod / local server
